packages/flame/test/components/position_component_test.dart

```diff
@@ -632,7 +632,8 @@ void main() {
           ..drawLine(const Offset(0, -2), const Offset(0, 2))
           ..drawLine(const Offset(-2, 0), const Offset(2, 0))
           ..drawParagraph(null, const Offset(-30, -15))
-          ..drawParagraph(null, const Offset(-20, 10)),
+          ..drawParagraph(null, const Offset(-20, 10))
+          ..translate(0, 0), // canvas.restore
```

Comment:
why is this needed?

Reply:
after the render cycle our implementation resets the camera, so this represents that change. we could ignore a final translate(0, 0) but I think it makes sense to add it to the tests as it does happen in reality